Makefile

```diff
@@ -1,4 +1,4 @@
-PACKAGE_NAME = osbuild-installer-backend
```

Comment:
I like this change! Would it makes sense to rename the whole project to `osbuild-installer` and drop the `-backend` suffix?

Reply:
it does sound a bit silly, I copied it from the cloud rh projects but yea I'm in favour of dropping it :)

Reply:
Hm I don't have the rights to change the name of this project. Could you rename it to drop the `-backend` ?